js/layout-javascript/simple-list-code.js

```diff
@@ -1205,6 +1205,20 @@ DynamicList.prototype.renderLoopHTML = function (iterateeCb) {
         requestAnimationFrame(render);
       } else{
         _this.$container.find('.simple-list-container').removeClass('loading').addClass('ready');
+
+        // SuperClamp works only when elemet has a fixed max-height or height
+        // To do so we need to take computed line-height and multiple it on data-line-clamp value
+        setClampHeight('[data-line-clamp]')
```

Comment:
Where is the `setClampHeight` function defined?

Reply:
@tonytlwu it is defined in the `build.js` file. I moved there to prevent code duplicate because we need it in both layouts.

Reply:
I don't see it in the `build.js` file. Did you commit the changes?

Also, you should use the `utils.js` file for commonly used functions in the LFD.

Reply:
@tonytlwu I've tried to add these functions to the `utils.js` but when I test it at the line that you highlighted function doesn't declare yet.
Is there any specific to add functions to this file?

I've added my functions into the body of the `Fliplet.Registry.set('dynamicListUtils', (function () {`

Reply:
@YaroslavOvdii I think I know why it doesn't work. but I can't comment on it unless you've committed your code.

Reply:
@tonytlwu I think I figure it out why it doesn't work earlier.
Added new commit.